process/coordinator/process.go

```diff
@@ -1374,7 +1493,11 @@ func getNumOfCrossShardScCallsOrSpecialTxs(
 }
 
 // VerifyCreatedMiniBlocks re-checks gas used and generated fees in the given block
-func (tc *transactionCoordinator) VerifyCreatedMiniBlocks(header data.HeaderHandler, body *block.Body) error {
+func (tc *transactionCoordinator) VerifyCreatedMiniBlocks(
+	header data.HeaderHandler,
+	body *block.Body,
+	processedMiniBlocks *processedMb.ProcessedMiniBlockTracker,
```

Comment:
same as for preprocessors, maybe add the tracker to the transactionsCoordinator and remove it as a parameter.

Reply:
As I explained above this will be a subject of a bigger refactor task which was added in Jira -> EN-12158 with a detailed description and all the steps which should be done